Description: Makefile for ubuntu
 Ubuntu needs to have Makefile in root directory
 .

---
--- /dev/null
+++ mtn-3.3/Makefile
@@ -0,0 +1,50 @@
+#CFLAGS=-W -Wall -O3 -s -D_FILE_OFFSET_BITS=64 -mtune=i686 -march=i386 -I/home/tuitfun/include -L/home/tuitfun/lib
+#CFLAGS=-W -Wall `pkg-config --cflags libavformat`
+
+#DESTDIR - reserved for rpmbuild
+UNAME=$(shell uname -s)
+CFLAGS=-W -Wall -O3
+OUT=bin
+LIBSDIR=lib
+
+ifeq ($(UNAME),FreeBSD)
+LIBS+=-L/usr/local/lib
+INCPATH+=-I/usr/local/include
+endif
+
+ifeq ($(UNAME),Linux)
+INCPATH=-I/usr/include/ffmpeg
+endif
+
+LIBS+=-lavcodec -lavformat -lavcodec -lswscale -lavutil -lgd -lfreetype -ljpeg -lz -lm -lpthread
+
+mtn: src/mtn.c outdir
+	$(CC) -o $(OUT)/mtn src/mtn.c $(INCPATH) $(CFLAGS) $(LIBS)
+
+outdir:
+	mkdir -p $(OUT)
+
+clean:
+	rm -f $(OUT)/mtn
+
+distclean:
+	rm -rf $(OUT)
+
+rebuild:  clean mtn
+
+
+install: mtn
+	mkdir -p $(DESTDIR)/usr/bin/
+	install -m 755 $(OUT)/mtn $(DESTDIR)/usr/bin/
+	mkdir -p $(DESTDIR)/usr/share/doc/mtn
+	cp -Rp doc/* $(DESTDIR)/usr/share/doc/mtn
+	cp -p gpl-2.0.txt $(DESTDIR)/usr/share/doc/mtn
+	gzip --force man/mtn.1
+	mkdir -p $(DESTDIR)/usr/share/man/man1
+	cp -p man/mtn.1.gz $(DESTDIR)/usr/share/man/man1/
+
+uninstall:
+	rm -f  $(DESTDIR)/usr/share/man/man1/mtn.1.gz
+	rm -f  $(DESTDIR)/usr/bin/mtn
+	rm -rf $(DESTDIR)/usr/share/doc/mtn
+
